core/services/ocr2/plugins/ocr2keeper/log_provider.go

```diff
@@ -34,15 +35,17 @@ func NewLogProvider(
 	lookbackBlocks int64,
 ) (*LogProvider, error) {
 	var err error
+	filterName := fmt.Sprintf("OCR2KeeperRegistry - %s", registryAddress)
```

Comment:
i can't tell from the code -- are these names persisted to the db? would it be problematic if they change from one release/version to another?

Reply:
Yes, they are persisted to db.  I think in generally we don't consider the db tables to be "user facing", unless things are read from there and displayed in the UI or console.  These strings would be in the tables, and could show up in logs.  We could still change them with an appropriate db migration, so I think it's okay.

We could make all of these random integers to avoid having to migrate the db if they change, but... I think I'd prefer the logs to be more readable, in case of any problems come up that require a rapid response.

Reply:
Would it make sense to provide a helper func or type to format these all in the same standard way, rather than relying on callers to follow the pattern? Would keep things aligned and make it simpler to refactor later if necessary too. For example:
```go
logpoller.FilterName("OCR2KeeperRegistry", registryAddress)

func FilterName(id string, args ...any) string {
	var s strings.Builder
	s.WriteString(id)
	s.WriteString(" - ")
	fmt.Fprintf("%s", args[0])
	for _, a := range args[1:] {
		fmt.Fprintf(":%s", a)
	}
	return s.String()
}
```